src/test/scala/com.snowplowanalytics.maxmind.iplookups/IpLookupsTest.scala

```diff
@@ -71,11 +69,11 @@ object IpLookupsTest {
         postalCode = Some("98354"),
         metroCode = Some(819),
         regionName = Some("Washington")
-      ).success.some,
-      "Century Link".success.some,
-      "Lariat Software".success.some,
-      new AddressNotFoundException("The address 216.160.83.56 is not in the database.").failure.some,
-      new AddressNotFoundException("The address 216.160.83.56 is not in the database.").failure.some
+      ).asRight.some,
+      "Century Link".asRight.some,
+      "Lariat Software".asRight.some,
+      new AddressNotFoundException("The address 216.160.83.56 is not in the database.").asLeft.some,
+      new AddressNotFoundException("The address 216.160.83.56 is not in the database.").asLeft.some
```

Comment:
why not `.right` and `.left`?

Reply:
there is no method `.right` and `.left`. or am I missing something?

Reply:
you're right my bad, I'm too used to scalaz